### PR TITLE
systemdgenie: init at  0.99.0

### PR DESCRIPTION
**Pull request by pasqui23**
###### Things done

- Built on platform(s)
  - [] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/system/systemdgenie/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    repo = pname;
+    owner = "system";
+    rev = "v${version}";
+    hash = "sha256-y+A2OuK1ZooPY5W0SsXEb1aaOAJ2b7QSwiumolmAaR4=";
+  };
+  nativeBuildInputs = [ cmake extra-cmake-modules wrapQtAppsHook ];
+  buildInputs = [ kxmlgui kdelibs4support ];
+  meta = with lib; {
+    description = "Systemd management utility";
+    homepage = "https://www.gnu.org/software/hello/manual/";
```
Comment (pennae) [inline pkgs/applications/system/systemdgenie/default.nix:24]: looks like you forgot to update this field from the template

File: pkgs/applications/system/systemdgenie/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+, extra-cmake-modules
+, kxmlgui
+, fetchFromGitLab
+, kdelibs4support
+, wrapQtAppsHook
+}:
+stdenv.mkDerivation rec{
+  pname = "SystemdGenie";
+  version = "0.99.0";
+  src = fetchFromGitLab {
```
Comment (SuperSandro2000) [inline pkgs/applications/system/systemdgenie/default.nix:13]:
```suggestion
}:

stdenv.mkDerivation rec{
  pname = "SystemdGenie";
  version = "0.99.0";

  src = fetchFromGitLab {
```

File: pkgs/applications/system/systemdgenie/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    domain = "invent.kde.org";
+    repo = pname;
+    owner = "system";
+    rev = "v${version}";
+    hash = "sha256-y+A2OuK1ZooPY5W0SsXEb1aaOAJ2b7QSwiumolmAaR4=";
+  };
+  nativeBuildInputs = [ cmake extra-cmake-modules wrapQtAppsHook ];
+  buildInputs = [ kxmlgui kdelibs4support ];
+  meta = with lib; {
```
Comment (SuperSandro2000) [inline pkgs/applications/system/systemdgenie/default.nix:22]:
```suggestion
    sha256 = "sha256-y+A2OuK1ZooPY5W0SsXEb1aaOAJ2b7QSwiumolmAaR4=";
  };

  nativeBuildInputs = [ cmake extra-cmake-modules wrapQtAppsHook ];

  buildInputs = [ kxmlgui kdelibs4support ];

  meta = with lib; {
```

File: pkgs/applications/system/systemdgenie/default.nix [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    description = "Systemd management utility";
+    homepage = "https://www.gnu.org/software/hello/manual/";
+    license = licenses.gpl;
+    maintainers = [ maintainers.pasqui23 ];
+    platforms = platforms.linux;
+  };
+
+}
```
Comment (SuperSandro2000) [inline pkgs/applications/system/systemdgenie/default.nix:30]:
```suggestion
  };
}
```

File: pkgs/applications/system/systemdgenie/default.nix [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    owner = "system";
+    rev = "v${version}";
+    hash = "sha256-y+A2OuK1ZooPY5W0SsXEb1aaOAJ2b7QSwiumolmAaR4=";
+  };
+  nativeBuildInputs = [ cmake extra-cmake-modules wrapQtAppsHook ];
+  buildInputs = [ kxmlgui kdelibs4support ];
+  meta = with lib; {
+    description = "Systemd management utility";
+    homepage = "https://www.gnu.org/software/hello/manual/";
+    license = licenses.gpl;
```
Comment (SuperSandro2000) [inline pkgs/applications/system/systemdgenie/default.nix:25]: What gpl?

File: pkgs/top-level/all-packages.nix [hunk 6]
```diff
@@ -9753,6 +9753,8 @@ with pkgs;
     inherit (xorg) xauth xorgserver;
   };
 
+  systemdgenie = libsForQt5.callPackage ../applications/system/systemdgenie/default.nix { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:9756]:
```suggestion
  systemdgenie = libsForQt5.callPackage ../applications/system/systemdgenie { };
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**